libs/coin-evm/docs/deviceTransactionConfig.md

```diff
@@ -0,0 +1,20 @@
+# `deviceTransactionConfig.ts`
+Set of functions responsible for providing to a UI client the different texts that are displayed on the screen of the nano for a specific transaction. This can only be considered as a *best effort* as there is no way to actually predict what the nano will display, as it could differ depending on the version of the nano app on the device or on the external plugins installed on the nano (Paraswap, 1inch, etc..)
+
+## Methods
+
+#### inferDeviceTransactionConfigWalletApi 
```

Comment:
Oh wow, wasn't aware of that. Maybe more parameters should be enforced when using a the ledger libarry :thinking: 

Anyway this is very important knowledge :+1:

Reply:
> Maybe more parameters should be enforced when using a the ledger libarry 🤔
Not sure to understand what you mean by that 🤔

Reply:
Do you think there is a way to improve wallet API calls to provide more context instead of decoding the data ?